src/modules/commander/PreflightCheck.cpp

```diff
@@ -389,7 +389,7 @@ static bool airspeedCheck(orb_advert_t *mavlink_log_pub, vehicle_status_s &statu
 	if ((orb_copy(ORB_ID(differential_pressure), fd_diffpres, &differential_pressure) != PX4_OK) ||
 	    (hrt_elapsed_time(&differential_pressure.timestamp) > 1_s)) {
 		if (report_fail && !optional) {
-			mavlink_log_critical(mavlink_log_pub, "PREFLIGHT FAIL: AIRSPEED SENSOR MISSING");
+			mavlink_log_critical(mavlink_log_pub, "Preflight Fail: Airspeed Sensor missing");
```

Comment:
Couple comments here:
1) this is the first use of "< > sensor missing".  The other sensors print out "no < > sensor".  From a user standpoint I like the "< > sensor missing" message better because it implies that the sensor is both expected to be there and also that it is missing.  The meaning for "no < > sensor" message is a bit more vague.  Either way, I think we should aim for consistency.
2) can this be extended to support multiple diffpres/airspeed sensors?

Reply:
1. Good point, I switched to the 'sensor missing' version.
2. I don't know, it's probably a bigger thing.

Reply:
2. Yes it can and yes it's a slightly bigger thing.